### PR TITLE
Fix code block editor popover styles

### LINKED ISSUE #18693: Code Block Interface popover item unreadable on Dark Mode
### Describe the Bug

Browsers tested:
- Firefox 113.0.1 (64-bit)
- Chrome 113.0.5672.126 (Official Build) (arm64)
- Safari 16.4 (18615.1.26.110.1)
OS:
- macOS 13.3.1 (a) (22E772610a).

When using Directus v10.1.1 in Dark Mode, using a Code Block interface, the popover items are not easily visible.

### To Reproduce

1. Set admin user to use dark mode.
2. Create new collection with code block interface
3. Add a new item to collection, select code block popover items.
4. See hard to read items.

Screenshot 2023-05-22 at 1 05 52 PM

### Directus Version

10.1.1

### Hosting Strategy

Self-Hosted (Docker Image)

### PR DESCRIPTION
**Pull request by that1matt**
Fixes: #18693 

light-theme
dark-theme

I believe this is small enough that it doesn't warrant a changeset, but happy to add a patch changeset if required.

### CODE REVIEW

**Review by paescuj — APPROVED** [review]
Thanks, looks good to me 🚀 
Let's add a changeset since it resolves an actual UI issue 👍

File: app/src/interfaces/input-block-editor/editorjs-overrides.css [hunk 1]
```diff
@@ -612,4 +621,4 @@
 
 .codex-editor .ce-inline-tool--active {
 	color: var(--primary);
-}
+}
```
Comment (licitdev) [inline app/src/interfaces/input-block-editor/editorjs-overrides.css:624]:
```suggestion
}

```
Should contain an empty line according to our prettier formatting

File: app/src/interfaces/input-block-editor/editorjs-overrides.css [hunk 2]
```diff
@@ -60,6 +60,14 @@
 	border-radius: var(--border-radius);
 }
 
+.codex-editor .ce-popover-item__title {
+	color: var(--foreground-normal);
+}
+
+.codex-editor .ce-popover-item:hover {
+	background: var(--v-list-background-color-hover) !important;
+}
```
[the remark this answers is not in the document]
Comment (licitdev) [reply app/src/interfaces/input-block-editor/editorjs-overrides.css:69]: Just checked out the `10.1.0` tag and the popover displays as shown in https://github.com/directus/directus/pull/18525#pullrequestreview-1421108108.
Seems like some recent changes caused this issue somehow. 🧐

[the remark this answers is not in the document]
Comment (paescuj) [reply app/src/interfaces/input-block-editor/editorjs-overrides.css:69]: Maybe due to https://github.com/directus/directus/pull/18643/files#diff-012b982f97a0d326aeec58dd3b789484b05a944d609afe1b8ed5e299fc485f61R44?

[the remark this answers is not in the document]
Comment (that1matt) [reply app/src/interfaces/input-block-editor/editorjs-overrides.css]: Got this fixed in my latest commit.

**Review by paescuj — APPROVED** [review]

**Review by licitdev — APPROVED** [review]

**Pull request merged**